openstack/manila/alerts/openstack/cert.alerts

```diff
@@ -0,0 +1,27 @@
+groups:
+- name: openstack-manila.alerts
+  - alert: OpenstackManilaCaCertificateExpireSoon
+    expr: cert_exporter_configmap_expires_in_seconds{issuer="SAP Global Root CA"} < 2419200
+    for: 1h
+    labels:
+      severity: info
+      support_group: compute-storage-api
+      tier: os
+      service: manila
+      context: ca
+    annotations:
+      description: CA certificate {{ $labels.cn }} will expire in 4 weeks or less.
+      summary: CA certificate {{ $labels.cn }} will expire.
+
+  - alert: OpenstackManilaCaCertificateExpireVerySoon
+    expr: cert_exporter_configmap_expires_in_seconds{issuer="SAP Global Root CA"} < 604800
+    for: 1h
+    labels:
+      severity: warn
+      support_group: compute-storage-api
```

Comment:
Please remove the support group or add a playbook ;)

Reply:
https://github.wdf.sap.corp/cc/documentation-operation/pull/2326

Reply:
I'm confused now, you added a playbook and removed the support group anyway?
Do you still want us (aka manila maintainers) to be the only ones to execute this?

In any case: can you add the link to the playbook, please?

Reply:
updated